### ISSUE TITLE
Adding F2 to evaluation metrics

### ISSUE THREAD

**Issue by jack-hillesheim** [autogluon/autogluon#4371]
## Description
Please add F2 as an evaluation metric. It is very useful when modeling with an emphasis on recall. Even better than F2 would perhaps be fbeta, which allows you to specify the degree to which recall is more important.

## References
https://scikit-learn.org/stable/modules/generated/sklearn.metrics.fbeta_score.html
